resources/data/elephpants.json

```diff
@@ -2,627 +2,939 @@
     "elephpants": [
         {
             "id": 1,
-            "name": "Original Blue",
-            "description": "First Blue",
-            "sponsor": "Nexen \/ Alter Way",
-            "year": 2007,
-            "image": "1-original-blue.jpg"
-        },
-        {
+			"name": "Original Blue",
```

Comment:
Spacing

Reply:
Should be resolved. Very odd behaviour...